### PR TITLE
Update the documents for the release 1.3.0

### PR DESCRIPTION
**Pull request by danfengliu**
Update the documents for the release 1.3.0

Signed-off-by: danfengl <danfengl@vmware.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,8 +302,8 @@ It is not possible to use different credentials for additional Backup Storage Lo
 
 ### Prerequisites
 
-* Velero 1.6.0 or later
-* AWS plugin must be installed, either at install time, or by running `velero plugin add velero/velero-plugin-for-aws:v1.2.1`
+* Velero 1.7.0 or later
```
Comment (ywk253100) [inline README.md:305]: This shouldn't be updated, the addition bsl with their own credential is supported from v1.6.0

File: README.md [hunk 2]
```diff
@@ -301,8 +302,8 @@ It is not possible to use different credentials for additional Backup Storage Lo
...
+* AWS plugin must be installed, either at install time, or by running `velero plugin add velero/velero-plugin-for-aws:v1.3.0`
```
Comment (ywk253100) [inline README.md:306]: Same here, maybe we can use the same style as https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure/pull/107 ?

File: changelogs/CHANGELOG-1.3.0.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+## All changes
+
+This release contains no user facing changes but includes fixes for Change the base image to distroless.
+CVE-2021-3121 and CVE-2021-3580
```
Comment (ywk253100) [inline changelogs/CHANGELOG-1.3.0.md:4]: Remove this line

File: changelogs/CHANGELOG-1.3.0.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+## All changes
+
+This release contains no user facing changes but includes fixes for Change the base image to distroless.
```
Comment (ywk253100) [inline changelogs/CHANGELOG-1.3.0.md:3]: `Change` -> `changing`

File: README.md (hunk 1 above)
Comment (danfengliu) [reply README.md:305]: Got it, thanks

File: changelogs/CHANGELOG-1.3.0.md (hunk 3 above)
Comment (danfengliu) [reply changelogs/CHANGELOG-1.3.0.md:4]: I'm curious about it that the PR for cve was just merged a few hours ago

**Review by ywk253100 — APPROVED** [review]

**Review by reasonerjt — APPROVED** [review]

**Pull request merged**